docs/advanced/typescript.mdx

```diff
@@ -15,8 +15,6 @@ improve the developer experience for TypeScript users.
 - `@mdx-js/runtime`
 - `@mdx-js/vue`
 - `remark-mdx`
-- `remark-mdx-remove-exports`
-- `remark-mdx-remove-imports`
```

Comment:
deprecated (ESM is just one node now: we could create a new package but it’s also a one-liner that folks can do manually)